.travis.yml

```diff
@@ -4,7 +4,7 @@ addons:
   apt:
     packages:
     - zsh
-bundler_args: --without production development --deployment --jobs=3 --retry=3
```

Comment:
the same amount of gems will be installed. now we just don't have any production gems anymore 😂